conf/locale/rtl/LC_MESSAGES/django.po

```diff
@@ -13735,6 +13704,46 @@ msgstr ""
 msgid "Your {platform_name} account email has been updated"
 msgstr "غخعق {platform_name} شذذخعرف ثوشهم اشس زثثر عحيشفثي"
 
+#: common/templates/student/edx_ace/accountactivation/email/body.html
+msgid "Hello"
+msgstr "اثممخ"
```

Comment:
I think I asked you this last year :)
What is the gibberish Arabic text in this file and the file above it?